### PR TITLE
spatial 0.4.2 (new cask)

### PR DESCRIPTION
**Pull request by mikeswanson**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for a stable version.
- [x] `brew audit --cask --online spatial` is error-free.
- [x] `brew style --fix spatial` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/s/spatial.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 "ba4b00e83983898dda7bd517ab481598267b25d4fbbe700f00d16c709a05ca21"
+
+  url "https://www.mikeswanson.com/spatial/releases/spatial_#{version}.zip"
+  name "spatial"
+  desc "Tool for working with MV-HEVC/spatial videos"
+  homepage "https://blog.mikeswanson.com/spatial"
+
+  livecheck do
+    url "https://www.mikeswanson.com/spatial/releases/"
+    regex(/href=.*?spatial_(\d+(?:\.\d+)+)\.zip/i)
```
Comment (bevanjkay) [inline Casks/s/spatial.rb:12]: Minor regex tweak to conform to prevailing standards.

```suggestion
    regex(/href=.*?spatial[._-]v?(\d+(?:\.\d+)+)\.zip/i)
```

File: Casks/s/spatial.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+cask "spatial" do
+  version "0.4.2"
+  sha256 "ba4b00e83983898dda7bd517ab481598267b25d4fbbe700f00d16c709a05ca21"
+
+  url "https://www.mikeswanson.com/spatial/releases/spatial_#{version}.zip"
+  name "spatial"
```
Comment (bevanjkay) [inline Casks/s/spatial.rb:6]:
```suggestion
  name "Spatial"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/s/spatial.rb (hunk 2 above)
Comment (mikeswanson) [reply Casks/s/spatial.rb:6]: Looks good.

File: Casks/s/spatial.rb (hunk 1 above)
Comment (mikeswanson) [reply Casks/s/spatial.rb:12]: Thanks.

**Review by mikeswanson — COMMENTED** [review]
All looks good to me.

**Review by bevanjkay — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**